sms-gateway.php

```diff
@@ -26,7 +26,10 @@
 <?php   }
     } 
     else if (str_exists(strtoupper($address), strtoupper('jft'))) {
-        $message = $client->messages->create($_REQUEST['From'], array("from" => $_REQUEST['To'], "body" => get_jft(true)));
+        $jft_chunks = get_jft(true);
+        for ($i = 0; $i < count($jft_chunks); $i++) {
+            $message = $client->messages->create($_REQUEST['From'], array("from" => $_REQUEST['To'], "body" => $jft_chunks[$i]));
```

Comment:
no need to return something here to `$message`

Reply:
gotcha